src/DataDryad.jl

```diff
@@ -0,0 +1,42 @@
+immutable DataDryad <: DataRepo
+end
+
+base_url(::DataDryad) = "https://datadryad.org/mn/"
+
+function description(::DataDryad, mainpage)
+    desc = replace(text_only(first(matchall(sel".article-abstract", mainpage.root))), "Abstract ", "")
+    author = text_only(first(matchall(sel".pub-authors a", mainpage.root)))
+    license = getattr(first(matchall(sel".single-image-link", mainpage.root)), "href")
+    date = now()
+    paper = text_only(first(matchall(sel".citation-sample a", mainpage.root)))
```

Comment:
Is it possible/easy to get the name of the paper and it's author list?

Ideally we would say something like:

`von Wettberg EJB, Chang PL, Başdemir F, Carrasquila-Garcia N, Korbu L, Moenga SM, Bedada G, Greenlon A, Moriuchi KS, Suryawanshi V, Cordeiro MA, Noujdina NV, Dinegde KN, Abbas Shah Sani SG, Getahun T, Vance L, Bergmann E, Lindsay D, Mamo BE, Warschefsky EJ, Dacosta-Calheiros E, Marques E, Yilmaz MA, Cakmak AM, Rose J, Migneault A, Krieg CP, Saylak S, Temel H, Friesen ML, Silor E, Akhmetov Z, Ozelik H, Kholova J, Can C, Gaur P, Yildirim M, Sharma H, Vadez V, Tesfaye K, Woldemedhin AF, Tar'an B, Aydogan A, Bukun B, Penmetsa RV, Berger J, Kahraman A, Nuzhdin SV, Cook DR (2018) Ecology and genomics of an important crop wild relative as a prelude to agricultural innovation. Nature Communications 9: 649. https://doi.org/10.1038/s41467-018-02867-z`

And similar for the dataset.

If it is not easy to get that into I am fine to merge as is with regards to this.

Reply:
Implemented.